src/shared/bus-unit-util.c

```diff
@@ -1401,6 +1401,14 @@ static int bus_append_path_property(sd_bus_message *m, const char *field, const
         return 0;
 }
 
+static int bus_append_scope_property(sd_bus_message *m, const char *field, const char *eq) {
+        if (streq(field, "TimeoutStopSec"))
+
```

Comment:
This empty newline here is not necessary... but I see that this matches the surrounding code. OK, let's keep it this way for now.